.github/workflows/main.yml

```
@@ -69,112 +74,173 @@
-          kubectl apply -f mongodb-service.yaml
-          kubectl apply -f backend-deployment.yaml
-          kubectl apply -f backend-service.yaml
-          kubectl apply -f frontend-deployment.yaml
-          kubectl apply -f frontend-service.yaml
+  cleanup:
+    needs: [deploy, rollback]
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)